Resources/Private/Templates/Content/Error.html

```diff
@@ -16,7 +16,7 @@
 						<h4 class="alert-title">Warning</h4>
 
 						<div class="alert-message">
-							<f:translate key="tt_content.tx_fed_fcefile.emptyMessageinBackend"/>
+							<f:translate extensionName="fluidcontent" key="tt_content.tx_fed_fcefile.emptyMessageinBackend"/>
```

Comment:
The `extensionName` is the `UpperCamelCase` version of `extension_key` - it's kind of confusing but it needs to be provided that way; if it works with this lowercase format then it is by coincidence (because our key doesn't contain an underscore, possibly case-insensitive filesystems) and could change at any time.